backend/src/services/teams/teams.repo.js

```diff
@@ -100,9 +143,25 @@ class TeamsRepo {
       throw e;
     }
 
+    const updateData = { ...data };
+
+    if (typeof updateData.name === 'string') {
+      updateData.name = updateData.name.trim();
+      if (updateData.name.length === 0) {
+        const e = new Error('name must not be empty');
+        e.code = 'BAD_REQUEST';
+        throw e;
+      }
+    }
+
+    if (typeof updateData.description === 'string') {
+      const trimmed = updateData.description.trim();
+      updateData.description = trimmed.length > 0 ? trimmed : null;
+    }
```

Comment:
[Non-blocking Suggestion]
Since the same input validation logic is used in create and update, we can extract this into helper functions.